ui/translations/en-us.yaml

```diff
@@ -37,34 +37,34 @@ page:
 form:
   project_new:
     title: 'New Project'
-    project_name_label: 'Project Name'
-    create_git_label: 'Connect a git repository'
-    create_git_helptext: 'If you skip this, you can always configure this later in project settings.'
-    create_git_checkbox_label: 'Connect a repository to this project'
-    button: 'Create project'
+    project_name_label: 'Project name'
+    create_git_label: 'Connect a Git repository'
+    create_git_helptext: 'If you skip this, you can always configure this later in Project settings.'
+    create_git_checkbox_label: 'Connect a repository to this Project'
+    button: 'Create Project'
   project_settings:
     git_title: 'Git settings'
-    git_source_url_label: 'Git Source URL'
-    git_source_ref_label: 'Git Ref'
-    git_source_ref_helptext: 'This can be a branch name, a tag name, or a fully qualified git ref such as'
+    git_source_url_label: 'Git source URL'
+    git_source_ref_label: 'Git ref'
+    git_source_ref_helptext: 'This can be a branch name, a tag name, or a fully qualified Git ref such as'
     git_source_ref_example: 'refs/pull/1014'
-    git_source_path_label: 'Git Source Path (optional)'
+    git_source_path_label: 'Git source path (optional)'
     git_auth_type_label: 'Authentication'
-    git_auth_type_basic_label: 'Username & Password'
-    git_auth_type_not_set_label: 'No Authentication'
+    git_auth_type_basic_label: 'Username & password'
+    git_auth_type_not_set_label: 'No authentication'
     git_auth_type_ssh_label: 'SSH'
     git_source_ssh_user_label: 'Git User (optional)'
     git_source_ssh_password_label: 'Passphrase (optional)'
     git_source_ssh_key_label: 'SSH Private Key'
     git_source_basic_username_label: 'Username'
     git_source_basic_password_label: 'Password'
-    git_sync_label: 'Automated Sync'
+    git_sync_label: 'Automated sync'
     git_sync_description_label: 'Enable automated deployment on changes'
     hcl_title: 'Waypoint.hcl settings'
-    hcl_location: 'Waypoint.hcl config location'
+    hcl_location: 'waypoint.hcl config location…'
```

Comment:
why the ellipsis here?

Reply:
@gregone In the UI the action buttons finish the sentence, so I think it makes sense to use the ellipsis to indicate that. I don't feel super strongly about it though.

![CleanShot 2021-06-08 at 14 33 02](https://user-images.githubusercontent.com/51724/121194233-792ace80-c866-11eb-90db-fa8ff0d7b7c3.png)

Reply:
What about something like this instead

> waypoint.hcl config location:

Reply:
@briancain 
> What about something like this instead

> >waypoint.hcl config location:

The main reason against it is consistency, no other labels use colons, so I'd avoid introducing a change in text patterns. 

@dizzyup I agree with that choice, removing the "sentence" style gets rid of the problem